query-service-impl/src/test/java/org/hypertrace/core/query/service/projection/ProjectionTransformationTest.java

```diff
@@ -92,6 +93,30 @@ void transformsBasicAliasProjection() {
             .blockingGet());
   }
 
+  @Test
+  void transformsBasicAliasProjectionWithSubpath() {
```

Comment:
Shouldn't this be in `AttributeExpressionSubpathExistsFilteringTransformationTest` and a corresponding `CONTAINS_KEY` filter be added ?

Reply:
No, this test isn't about `CONTAINS_KEY` filters, those tests already exist in the class you referenced. This test is verifying projection works when using a subpath.